rest/admin_api.go

```diff
@@ -105,12 +105,15 @@ func (h *handler) handleCreateDB() error {
 		// store the cas in the loaded config after a successful insert
 		h.server.dbConfigs[dbName].cas = cas
 	} else {
-		if err := config.setup(dbName, h.server.config.Bootstrap, nil); err != nil {
+		if err := config.setup(dbName, BootstrapConfig{}, nil); err != nil {
```

Comment:
Can we add a comment here to clarify that we are intentionally not inheriting from the bootstrap when processing a legacy config.